i18n/en.i18n.json

```diff
@@ -185,6 +185,8 @@
   "LDAP_Dn" : "Distinguished Name (DN)",
   "LDAP_Enable" : "Enable LDAP",
   "LDAP_Port" : "LDAP Port",
+  "LDAP_Sync_User_Data" : "Keep user data in sync with server",
+  "LDAP_Sync_User_Data_FieldMap" : "User data field map",
```

Comment:
Could you start adding examples to the expected format and a more verbose explanation.
eg: 
>User data field map allows configuration how to populate user account fields (like email) from a record in ldap (once found).
>As an example, `{"cn":"name", "mail":"email"}` will choose a persion's human readable name from the `cn` attribute, and their email from the `mail` attribute. Available fields include name, and email.

Maybe this could be dynamically generated from the `config_server.coffee` types?